src/osd/OSD.cc

```diff
@@ -6911,10 +6911,18 @@ OSDService::ScrubJob::ScrubJob(CephContext* cct,
     double r = rand() / (double)RAND_MAX;
     sched_time +=
       scrub_min_interval * cct->_conf->osd_scrub_interval_randomize_ratio * r;
+    if (sched_time < timestamp) {
```

Comment:
can we update the implementation of `utime_t& operator+=(utime_t& l, double f)` instead? so if the result will overflow, we can just assign `std::limits<uint32_t>::max()` to it?

Reply:
right, file changed, help review,please